app/sprinkles/account/src/Database/Migrations/v400/GroupsTable.php

```diff
@@ -34,7 +34,7 @@ public function up()
                 $table->string('slug');
                 $table->string('name');
                 $table->text('description')->nullable();
-                $table->string('icon', 100)->nullable(false)->default('fa fa-user')->comment('The icon representing users in this group.');
+                $table->string('icon', 100)->nullable(false)->default('fas fa-user')->comment('The icon representing users in this group.');
```

Comment:
This will need to be done in a new migration.

Reply:
The `fa` tag can still be used but documentation states it has been deprecated. Though I could not find anything definitive on when that tag will actually stop working.

Reply:
Will still need to update this separately (and I think update the fast-track migration hidden somewhere) for this to properly take.

Reply:
Yeah that might be problematic. A new migration for sure (otherwise existing setup won't have the change). At one point the default could be removed altogether from the db structure and be done in the controller.

Reply:
> At one point the default could be removed altogether from the db structure and be done in the controller.

This would allow the use of other icon packages if for some reason the use of Font Awesome was not wanted at all.

Reply:
Yes indeed !

Reply:
It looks like this is already being set in the controller here? 
https://github.com/userfrosting/UserFrosting/blob/master/app/sprinkles/admin/src/Controller/GroupController.php#L386

Reply:
Indeed. While it would be better to set the default in the model, I don't see why a default value must be defined in the first place outside the controller.